src/tsx/liveScreen.tsx

```diff
@@ -30,23 +32,28 @@ class LiveScreen extends React.PureComponent<{ coreConnection: ICoreConnection }
     return (
       <div className={style.screen}>
         <StatusView/>
-        <Timeline/>
+        <Timeline nodes={this.state.nodes} currentNode={this.state.currentNode}/>
         <PdfViewer/>
       </div>
     );
   }
 
-  private registerCallbacks() {
-    this.props.coreConnection.addEventListener("increment", () => {
-      this.setState({ times: this.state.times + 1 });
+  private initConnection() {
+    this.props.coreConnection.addEventListener("nodes", (event) => {
```

Comment:
Would be nice to use the newly added constant variable here for "nodes". Also for "current-node" below. Is that possible (with scopes and such)?

Reply:
It's possible if we put them in a common costs-file, but I like seeing the literals so I can tell at a glance that it's an event name. Typescript will catch any typos here.

Reply:
Sounds fair to me 👌